### DISCUSSION TITLE
dialog的ShowCustomModal不能多层弹窗

### DISCUSSION THREAD

**Discussion by shucaitech** [irihitech/Ursa.Avalonia#461] [Q&A]
在dialog通过ShowCustomModal弹出的窗口中，继续使用ShowCustomModal弹出的子窗体不再是模态模式

**Comment by rabbitism** [comment]
你需要找到这个弹窗作为parent，否则弹窗只能遮挡main window

**Reply by fangjiajiajia** [reply]
showcustiommodal 输入参数只有一个hostId  已知窗口 怎么传入呢
